files/pulseaudio-bluetooth.conf

```diff
@@ -0,0 +1,7 @@
+<busconfig>
+  
+ <policy user="pulse">  
+  <allow send_destination="org.bluez"/>  
```

Comment:
This should probably be configurable for when someone wants to use another Bluetooth manager.

Reply:
pulseaudio-bluetooth is currently the only bluetooth manager which this role can install. Do you know any other a2dp bluetooth managers for linux?

Reply:
Okay I looked for it, the manager I had in mind also uses bluez.